src/DweetCard.tsx

```diff
@@ -102,6 +102,9 @@ export const DweetCard: React.FC<Props> = (props) => {
 
   useEffect(() => {
     const handler = (event: MessageEvent) => {
+      if (event.source !== iframe?.contentWindow) {
```

Comment:
Could be merged in a single `if` with 3 conditions, but it works perfectly anyway 👍🏼

Reply:
Yep. Kept them separate to make the code as straightforward as possible :)